webonary-cloud-api/lambda/getDictionary.ts

```diff
@@ -0,0 +1,42 @@
+import { APIGatewayEvent, Context, Callback } from 'aws-lambda';
+import { MongoClient } from 'mongodb';
+import { connectToDB } from './mongo';
+import { DB_NAME, COLLECTION_DICTIONARIES, COLLECTION_ENTRIES, DictionaryData } from './db';
+import * as Response from './response';
+
+let dbClient: MongoClient;
+
+export async function handler(
```

Comment:
Since this is the exported function it doesn't *need* a more descriptive name, but it could still help maintainers don't you think?

Reply:
This seems to be a standard with Lambda functions. Also, in the cdk stack.ts code, I can use a standard function using this common naming scheme:

```
function defaultLambdaFunctionProps(functionName: string): lambda.FunctionProps {
  const props: lambda.FunctionProps = {
    functionName,
    runtime: lambda.Runtime.NODEJS_12_X,
    code: new lambda.AssetCode('lambda'),
    handler: `${functionName}.handler`,
    timeout: cdk.Duration.seconds(60),
  };
  return props;
}
```